### DISCUSSION TITLE
Event based achievement  update

### DISCUSSION THREAD

**Discussion by dorukarslan** [Schweinepriester/github-profile-achievements#55] [Q&A] [answered]
Will there be any event based achievement this year? If there will be, is it happening regularly every year?

**Comment by alpkeskin** [comment, marked as answer]
As far as I know, currently there is no valid information about it. But you need to reasearch more to be sure.

**Reply by Schweinepriester** [reply]
I see you guys from the same company farming Galaxy Brain ;)
